pkcs7/Cargo.toml

```diff
@@ -17,7 +17,8 @@ rust-version = "1.65"
 [dependencies]
 der = { version = "0.7", features = ["oid"], path = "../der" }
 spki = { version = "0.7", path = "../spki" }
-x509-cert = { version = "0.2", default-features = false, path = "../x509-cert" }
+# x509-cert = { version = "0.2", default-features = false, path = "../x509-cert" }
+x509-cert = { version = "0.2", path = "../x509-cert" }
```

Comment:
It's this change that's activating `std`.

Instead you should activate the specific features you need.

Reply:
Thank's. I activated PEM handling for the added test to see, if I can add certificates (using existing example cert.pem). CI is green now.